Source/Core/GUI_Components/PawnIcons.cs

```diff
@@ -10,84 +10,84 @@ namespace PrisonLabor.Core.GUI_Components
     [StaticConstructorOnStartup]
     public class PawnIcons : MapComponent
```

Comment:
Does this icon really hit performance hard?

Reply:
Yeah it does. Well you can use the analyzer and patch the draw method.